rust/test_fracpack/tests/test.rs

```diff
@@ -336,7 +337,6 @@ fn t1() -> Result<()> {
         let unpacked = OuterStruct::unpack(&packed[..], &mut 0)?;
         assert_eq!(*t, unpacked);
         test_fracpack::bridge::ffi::round_trip_outer_struct(i, &packed[..]);
-        // TODO: optionals after fixed-data portion ends
```

Comment:
I believe this is not necessary, since all the structs that we are comparing with C++ examples has trailing optionals and we make sure they are equal. LMK if you think otherwise.